spec/features/admin_creates_a_product_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "Admin creates a proudct" do
```

Comment:
@desi That's the next PR I have coming. I implemented the authentication in that one.